### PR TITLE
Moving measurement to core

### PR DESCRIPTION
**Pull request by Peyman-N**
This pull request moves the `measurement` schema from the openMINDS-ephys to the openMINDS-core. 
Measurement schema is meant to capture a single and isolate measurement. More complex measurements should be handelle by core/recording see #272

### CODE REVIEW

File: schemas/data/measurement.schema.tpl.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                "https://openminds.ebrains.eu/ephys/ElectrodeArray",
+                "https://openminds.ebrains.eu/ephys/Electrode"
+            ]
+        },
+        "description": {
+            "type": "string",
+            "_instruction": "Enter a description for this measurement"
+        }
+    }
+}
```
Comment (lzehl) [inline schemas/data/measurement.schema.tpl.json]: There needs to be a newline at the end of JSONs.

File: schemas/data/measurement.schema.tpl.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            "_linkedTypes": [
+                "https://openminds.ebrains.eu/controlledTerms/MeasuredQuantity"
+            ]
+        },
+        "device": {
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "_instruction": "Device used for this measurement",
+            "_linkedTypes": [
```
Comment (lzehl) [inline schemas/data/measurement.schema.tpl.json]: Reduce to type /core/Device (reason will become clear with feedback for /core/Device PR #270 )

**Review by lzehl — CHANGES_REQUESTED** [review]
@UlrikeS91 and @lzehl reviewed together

File: schemas/data/measurement.schema.tpl.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+    "_type": "https://openminds.ebrains.eu/core/Measurement",
+    "required": [
+        "value",
+        "timestamp",
+        "measurementType"
```
Comment (lzehl) [inline schemas/data/measurement.schema.tpl.json]: this is not the correct name of the property. I think you mean "measuredQuantity" correct?

**Review by lzehl — CHANGES_REQUESTED** [review]

File: schemas/data/measurement.schema.tpl.json (hunk 3 above)
Comment (Peyman-N) [reply schemas/data/measurement.schema.tpl.json]: Yes

File: schemas/data/measurement.schema.tpl.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            "type": "string",
+            "_instruction": "Date and time at which this measurement was made."
+        },
+        "measuredQuantity": {
+            "_instruction": "Enter the quantity that is being measured.",
+            "_linkedTypes": [
+                "https://openminds.ebrains.eu/controlledTerms/MeasuredQuantity"
+            ]
+        },
+        "device": {
```
Comment (lzehl) [inline schemas/data/measurement.schema.tpl.json]: needs to link to the all possible deviceUsage schemas. (-> intro of category: deviceUsage)

File: schemas/data/measurement.schema.tpl.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "_instruction": "Device used for this measurement",
+            "_linkedTypes": [
+                "https://openminds.ebrains.eu/core/Device"
+            ]
+        },
+        "description": {
```
Comment (lzehl) [inline schemas/data/measurement.schema.tpl.json]: change to "additionalRemarks"

**Review by lzehl — APPROVED** [review]

**Review by lzehl — APPROVED** [review]

**Review by UlrikeS91 — APPROVED** [review]

**Review by apdavison — APPROVED** [review]

**Pull request merged**